### DISCUSSION TITLE
[v2] IndexedDB path changed after upgrading to tauri 2

### DISCUSSION THREAD

**Discussion by hut36** [tauri-apps/tauri#11231] [Q&A]
Hi everyone!

After upgrading to tauri 2 (referred to as new app below), I found that I'd "lost" all my app data, which was stored in IndexedDB. After some digging, it turns out that the IndexedDB path changed from `https_tauri.localhost_0.indexeddb.leveldb` to `http_tauri.localhost_0.indexeddb.leveldb`. The old `https_tauri.localhost_0.indexeddb.leveldb` is still there, and the app before upgrading to tauri 2 (referred to as old app below) can see the data.

To maintain backward compatibility, I need the new app to read/write dir `https_tauri.localhost_0.indexeddb.leveldb`. The question is, how? Another question is, renaming `https_tauri.localhost_0.indexeddb.leveldb` (accessed by the old app) to `http_tauri.localhost_0.indexeddb.leveldb` (accessed by the new app) doesn't make the new app see the data.

This is Windows 11, the full paths of mentioned dirs are:

- `C:\Users\[username]\AppData\Local\com.example.app\EBWebView\Default\IndexedDB\http_tauri.localhost_0.indexeddb.leveldb`
- `C:\Users\[username]\AppData\Local\com.example.app\EBWebView\Default\IndexedDB\https_tauri.localhost_0.indexeddb.leveldb`

**Comment by FabianLars** [comment]
Looks like we forgot to port the dangerousUseHttpScheme setting after all. I guess that's mostly my fault... Would you mind opening a normal GitHub issue for this instead so the team can hopefully quickly follow up (I am out of office right now)?

**Reply by hut36** [reply]
Done, https://github.com/tauri-apps/tauri/issues/11252. Seems I can't add label `platform: Windows`.

**Comment by hut36** [comment]
Sure, will do.
